tests/helpers.py

```diff
@@ -380,21 +388,34 @@ def delete_cephblockpool():
         kind=constants.CEPHBLOCKPOOL,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    pool_list = get_cephblockpool_name()
-    for item in pool_list:
-        logger.info(f"Deleting CephBlockPool with name {item}")
-        assert POOL.delete(resource_name=item)
+    logger.info(f"Deleting CephBlockPool with name {cbp_name}")
+    assert POOL.delete(resource_name=cbp_name)
     return True
 
 
-def create_cephfilesystem():
+def create_cephfilesystem(override_fs=True):
```

Comment:
please verify no one else is using this function as you changed its signature

Reply:
Checked no one else using this function